### ISSUE TITLE
CN TODO: expose 'revision' as argument

### ISSUE THREAD

**Issue by CodyCBakerPhD** [catalystneuro/IBL-to-nwb#64]
In all camera related interfaces, expose control over the explicit 'revision' value used (latest might not be the current 'released' or approved ones for use)

https://github.com/catalystneuro/IBL-to-nwb/blob/main/ibl_to_nwb/datainterfaces/iblposeestimationinterface.py#L44

**Comment by CodyCBakerPhD** [comment]
@grg2rsr Can you please confirm if the pose estimation data (DLC applied to camera data) is the only data stream for which exposing the 'revision' argument is needed?

Related PR #86

**Comment by grg2rsr** [comment]
no, this should apply to all processed data. Essentially, we want to be able to tell the ONE object which data of which revision to get.

**Comment by CodyCBakerPhD** [comment]
> Essentially, we want to be able to tell the ONE object which data of which revision to get.

So it should be passed to essentially any call of `one.load_object`?

> no, this should apply to all processed data.

Including wheel?

Spike sorting?

Atlas?

Can you show a quick snippet of it for the SpikeSortingLoader?

**Comment by grg2rsr** [comment]
I doubt that the wheel data is going to change from here on, but the spike sorting might be repeated as spike sorting algorithms will continue to be developed.

the `load_spike_sorting` function of the `SpikeSortingLoader` should simply support `revision` as an argument [link](https://github.com/int-brain-lab/ibllib/blob/3e807947523108e9db94c5ead34c56b487443d9b/brainbox/io/one.py#L1009C61-L1009C69).

I will get back to you regarding the atlas

**Comment by h-mayorquin** [comment]
The conversion pipeline now uses explicit revision constants throughout. For example, `BWM_REVISION = "2025-05-06"` is used consistently for the `SpikeSortingLoader` and other data loaders. The old file paths referenced in this issue no longer exist; the codebase has been restructured and the revision is now a controlled parameter in the conversion session logic. This is addressed in PR #108.
